cmake/modules/DetectArchitecture.cmake

```diff
@@ -11,6 +11,8 @@ elseif (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm32")
     set(ARM32 ON)
 elseif (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm64")
```

Comment:
I think you can replace the `arm3264` strings by `aarch32/64`.

Reply:
hmmm true, looks like it refers to the same thing (based on: https://unix.stackexchange.com/questions/461179/what-is-the-difference-between-different-implemetation-of-arm64-aarch64-for-linu)

Reply:
done :+1:

Reply:
maybe @tklengyel knows more about ARM ?

Reply:
true, I tried with my Hikey960 board and it works, maybe @tklengyel  could try with the other boards.